App/__tests__/AddUserToGroup.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render } from "@testing-library/react";
```

Comment:
Nice work testing your pages

Reply:
Tests are running into an error at present because of how the button component was implemented, will need to declare the types for all it's attributes individually which I'm unsure about the specifics of, any advice?

Reply:
I'm really not very familiar with frontend testing patterns.